### PR TITLE
[v7r3] docs: added better docs for JobShare and priorities

### PR DESCRIPTION
**Pull request by fstagni**
BEGINRELEASENOTES

*docs
FIX: added better docs for JobShare and priorities

ENDRELEASENOTES

### CODE REVIEW

File: docs/source/AdministratorGuide/Systems/WorkloadManagement/JobPriorities/index.rst [hunk 1]
```diff
@@ -46,8 +46,40 @@ based on the group and user it belongs to:
    * If it does *NOT*, it will get 1/(N*U) being U the number of users in the group
      with waiting jobs and N the number of *TaskQueues* of that user/group combination.
 
-On top of that users can specify a "UserPriority" to their jobs. To reflect that,
-DIRAC modifies the *TaskQueues* priorities depending on the "UserPriority" of the
+Administrators can set a different job shares depending on the user's group that runs the jobs,
```
Comment (atsareg) [inline docs/source/AdministratorGuide/Systems/WorkloadManagement/JobPriorities/index.rst:49]:
```suggestion
Administrators can set different job shares depending on the user's group that runs the jobs,
```

**Review by atsareg — APPROVED** [review]

**Review by andresailer — CHANGES_REQUESTED** [review]
Some table of content tree needs an update after the rename of the file.

And a couple of typos/grammar

```
/home/docs/checkouts/readthedocs.org/user_builds/dirac/checkouts/5928/docs/source/AdministratorGuide/Systems/WorkloadManagement/index.rst:80: WARNING: toctree contains reference to nonexisting document 'AdministratorGuide/Systems/WorkloadManagement/JobPriorities'
/home/docs/checkouts/readthedocs.org/user_builds/dirac/checkouts/5928/docs/source/DeveloperGuide/Internals/index.rst:5: WARNING: toctree contains reference to nonexisting document 'AdministratorGuide/Systems/WorkloadManagement/JobPriorities/index'
looking for now-outdated files... none found
pickling environment... done
checking consistency... /home/docs/checkouts/readthedocs.org/user_builds/dirac/checkouts/5928/docs/source/AdministratorGuide/Systems/WorkloadManagement/JobsPriorities.rst: WARNING: document isn't included in any toctree
```

**Comment by andresailer** [comment]
Still one warning
```
/home/docs/checkouts/readthedocs.org/user_builds/dirac/checkouts/5928/docs/source/DeveloperGuide/Internals/index.rst:5: WARNING: toctree contains reference to nonexisting document 'AdministratorGuide/Systems/WorkloadManagement/JobPriorities/index'
```

**Comment by fstagni** [comment]
I guess OK to merge now!

**Pull request merged**